packages/docs-page/server/loaders/remote-content.ts

```diff
@@ -79,22 +97,18 @@ const cachedFetchVersionMetadataList = moize(
 export function mapVersionList(
   list: VersionMetadataItem[]
 ): VersionSelectItem[] {
-  const versions = list
-    .sort((a, b) =>
-      semver.compare(semver.coerce(b.version)!, semver.coerce(a.version)!)
-    )
-    .map((e) => {
-      const { isLatest, version, display } = e
-
-      const displayValue = display || version
+  const versions = list.map((versionOption) => {
+    const { isLatest, version, display, releaseStage } = versionOption
+    const displayValue = display || version
 
-      return {
-        name: isLatest ? 'latest' : version,
-        label: isLatest ? `${displayValue} (latest)` : displayValue,
-        isLatest: isLatest || false,
-        version,
-      }
-    })
+    return {
+      name: isLatest ? 'latest' : version,
+      label: determineLabel(versionOption, displayValue),
+      isLatest: isLatest || false,
+      releaseStage: releaseStage as ReleaseStage,
```

Comment:
❔ is this `as ReleaseStage` type-casting still necessary? It _looks like_ the type should be correct even with out the `as ...`.